mmv1/products/apigee/terraform.yaml

```diff
@@ -47,11 +47,47 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       min_version: beta
       # Resource creation race
       skip_vcr: true
-
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/apigee_organization.go.erb
       encoder: templates/terraform/encoders/apigee_organization.go.erb
-
+  Instance: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{org_id}}/instances/{{name}}", "{{org_id}}/{{name}}"]
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_instance_basic"
+      skip_test: true
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_instance_basic_test"
+      primary_resource_id: "apigee_instance"
+      test_env_vars:
+        org_id: :ORG_ID
+        billing_account: :BILLING_ACCT
+      skip_docs: true
+      # Resource creation race
```

Comment:
Could you elaborate on this? If there is a resource creation race, I think that shouldn't impact VCR tests, since they don't run off real API requests.

Reply:
It looks like this also prevents the new tests from getting run on this PR so I would definitely prefer to not mark these as skip_vcr unless absolutely necessary.

Reply:
I just copied over this from the org tests (since this includes the org tests). It was introduced [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/4401/files#diff-d03f223d530588d47ff6aa088a55e79abf68090146cb8954fbf036a8381a40daR30), perhaps @slevenick can comment on the reasoning behind it?

Reply:
I removed it for the time being until we hear back from @slevenick if what was the reason for introducing it for Orgs (or use a boostrapped Org instead).

Reply:
You will need to add it back.

In that test we enable multiple `google_project_service` in a single terraform apply. These are not actually individual resources but sub-resources on a single actual resource. The problem is that Terraform does not apply these in a deterministic order, so on one run it could add service 1 first, then 2 then 3. Recorded requests will then only work if the resources are created in that order. If another run in VCR mode tries to create 2 then 3 then 1 it will fail with an inconsistent provider message

Reply:
That makes sense - thanks, Sam!

Reply:
I added skip_vcr back.